commands/upgrade/cluster/command.go

```diff
@@ -113,6 +116,7 @@ func initFlags() {
 	Command.ResetFlags()
 
 	Command.Flags().BoolVarP(&flags.Force, "force", "", false, "If set, no interactive confirmation will be required (risky!).")
+	Command.Flags().StringVarP(&flags.Release, "release", "", "", "The target release version for the upgrade. If no version is specified, the latest production version will be used.")
```

Comment:
Not a blocker, just a note: "production version" is a term that could raise some question marks. In fact we haven't settled yet for a term for this. See https://github.com/giantswarm/giantswarm/issues/14171

Reply:
Now that I read it again, this is false.
If no version is specified, the next "production" release will be used.
Let me rephrase it.